src/euidSdk.ts

```diff
@@ -58,6 +58,11 @@ export function assertEUID(sdk: typeof window.__euid): asserts sdk is EUID {
 }
 
 export function __euidInternalHandleScriptLoad() {
+  if ((window.__euid as EUID)?.initComplete && (window.__euid as EUID)?.initComplete()) {
```

Comment:
I don't think we care about _initComplete_. If we already had an instance of the class on _window.__euid_ but it hadn't been inited yet, this would create a second instance and any callbacks registered before that would end up called by both instances.

Reply:
My concern was exactly that.  Would it be a bad thing if the callbacks happened multiple times?  Using our example site, we create the __euid object on window and add a callback to it.  If we don't check initComplete, every script inclusion would call everything in the callbacks() array.  If this is harmless, cool.  But it seemed less ideal.  And maybe less future-proof.